<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢迎使用！</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>861</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>310</y>
       <width>111</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>分辨率</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_6">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(0, 0, 0)</string>
        </property>
        <item>
         <property name="text">
          <string>1080P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4K</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>111</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>硬盘</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_5">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(0, 0, 0)</string>
        </property>
        <item>
         <property name="text">
          <string>512G SSD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1TB SSG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1TB HDD</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>510</y>
       <width>121</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>键盘类型</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_10">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(0, 0, 0)</string>
        </property>
        <item>
         <property name="text">
          <string>机械键盘</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>薄膜键盘</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>210</y>
       <width>111</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>内存</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_3">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(0, 0, 0)</string>
        </property>
        <item>
         <property name="text">
          <string>4G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>64G</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>420</y>
       <width>111</width>
       <height>53</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_8">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(0, 0, 0)</string>
        </property>
        <item>
         <property name="text">
          <string>ABS塑料</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>合金材料</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>A面材质</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>121</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>屏幕尺寸</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_4">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(0, 0, 0)</string>
        </property>
        <item>
         <property name="text">
          <string>13.3英寸</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14.0英寸</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15.6英寸</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17.3英寸</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>电脑参数选择</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>210</y>
       <width>111</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>GPU容量</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_2">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(0, 0, 0)</string>
        </property>
        <item>
         <property name="text">
          <string>集成显卡</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8G</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>197</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>请将预期的电脑参数填写完整</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_10">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>420</y>
       <width>111</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>颜色</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_9">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(0, 0, 0)</string>
        </property>
        <item>
         <property name="text">
          <string>银白色</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黑色</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>水蓝色</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>红黑色</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>粉红色</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_11">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>刷新率</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_7">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(0, 0, 0)</string>
        </property>
        <item>
         <property name="text">
          <string>60Hz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>144Hz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>240Hz</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>590</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
background-color:rgb(24, 144, 255)</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>211</y>
       <width>121</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>CPU</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox">
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(0, 0, 0)</string>
        </property>
        <item>
         <property name="text">
          <string>Intel i5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Intel i7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Intel i9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AMD R5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AMD R7</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>241</width>
      <height>719</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(65, 147, 255);
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>     非常好电脑



















</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>861</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>771</width>
       <height>421</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>订单号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户电话</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>商品编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>下单时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>订单状态</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="layoutWidget_12">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>197</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="3" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>双击订单查看详情</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_18">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>订单列表</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>580</y>
       <width>371</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_12">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>输入订单号以查询</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>181</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>我的订单</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>推荐配置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>预测场景</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>入库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>待购表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>订单处理</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>620</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出系统</string>
    </property>
   </widget>
   <zorder>label_14</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>pushButton_8</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
